evm/src/cpu/kernel/tests/receipt.rs

```diff
@@ -119,7 +127,7 @@ fn test_receipt_encoding() -> Result<()> {
     // Get the expected RLP encoding.
     let expected_rlp = rlp::encode(&rlp::encode(&receipt_1));
 
-    let initial_stack = vec![retdest, 0.into(), 0.into()];
+    let initial_stack: Vec<U256> = vec![retdest, 0.into(), 0.into()];
```

Comment:
Those changes seem unnecessary, but they silenced the clippy warning of `needless vec` (which is actually needed because of the `Interpreter::new_with_kernel` expected parameter types).